docs/pim/create_custom_name_schema_strategy.md

```diff
@@ -0,0 +1,25 @@
+---
+description: Create CustomNameSchemaStrategy to generate URL aliases based on attribute values.
+---
+
+# Create custom Name Schema strategy
+
+Make sure you have properly configured attributes. Each attribute that you want to include in the URL alias must have a name schema strategy.
+
+## Create converting class
+
+Start by creating a `CustomNameSchemaStrategy` class, which implements `\Ibexa\ProductCatalog\NameSchema\NameSchemaStrategyInterface`. This class is responsible for converting attribute values into a string of URL parameters:
```

Comment:
If `\Ibexa\ProductCatalog\NameSchema\NameSchemaStrategyInterface` is extension point (it should be) then it needs to be moved to contracts // cc @kisztof